README.md

```diff
@@ -4,10 +4,11 @@ The leaderboard website displays scores submitted by different players. It also
 
 ## Live Demo
 
-[Not available yet]()
+[Click me](https://mov305.github.io/Leaderboard/dist/)
```

Comment:
- [ ] Remove the `Portfolio` title since this is not that project
![image](https://user-images.githubusercontent.com/62705947/186727576-55dbaafc-65dd-4cc1-bc3f-36d65ee3a0ea.png)